frontend/src/components/rules/rules.tsx

```diff
@@ -81,7 +81,7 @@ const Rules: React.FC = () => {
       {isOpen && (
         <Component>
           {glossary.map((item, index) => (
-            <Text key={index} title={item.title} text={item.text} />
+            <Text key={index} title={item.title} text={item.text} icons={item?.icons}/>
```

Comment:
key={index} https://react.dev/learn/rendering-lists 
<img width="1020" alt="image" src="https://github.com/user-attachments/assets/bc4ecdaf-758c-4a4a-a2e4-fddaf495ed15" />


```suggestion
{glossary.map(({title, text, icons, id}, index) => (
            <Text key={id} title={title} text={text} icons={icons}/>
```